models/FanPage.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  {
+    author: [{
+      type: mongoose.Types.ObjectId,
+      ref: 'User'
+    }],
+    artist: {
+      type: String,
+      required: true,
+    },
+    page_title: {
+      type: String,
+      required: true,
+    },
+    page_bio: {
+        type: String,
+        required: false,
+    },
+    user_tracks: {
```

Comment:
For all the variables we want to use camel case for our application ex: userTracks

In ERD we use snake case